code-challenges.test.js

```diff
@@ -23,10 +50,84 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 
 // b) Create the function that makes the test pass.
 
+const colors = (array) =>  {
+    return array.map(value => value.shuffle)
```

Comment:
Unfortunately, Javascript does not have a shuffle method that does this for us.  The more common approach is using sort and passing in an argument that allows you to adjust how it is sorted.  Also, keep in mind that you will need to remove the first element in the array first.